seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -324,9 +324,15 @@
       "id": "a2f1d72d9b908d0bd72bb9f6",
       "title": "Make a Person",
```

Comment:
Traducción sugerida:

"Completa el constructor de objetos con los métodos especificados a continuación:",
"getFirstName()",
"getLastName()",
"getFullName()",
"setFirstName(first)",
"setLastName(last)",
"setFullName(firstAndLast)",
"Ejecuta las pruebas para ver el resultado esperado de cada método.",
"Las funciones que aceptan argumentos deben aceptar sólo uno, y este tiene que ser una cadena.",
"Recuerda utilizar <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Leer-Buscar-Preguntar</a> si te sientes atascado. Intenta programar en pareja. Escribe tu propio código."